compiler/control/OMROptions.cpp

```diff
@@ -732,6 +733,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"enableSpecializedEpilogues",         "O\tapply restore non-volatiles selectively",         SET_OPTION_BIT(TR_EnableSpecializedEpilogues), "F"},
    {"enableTailCallOpt",                  "R\tenable tall call optimization in peephole", SET_OPTION_BIT(TR_EnableTailCallOpt), "F"},
    {"enableThisLiveRangeExtension",       "R\tenable this live range extesion to the end of the method", SET_OPTION_BIT(TR_EnableThisLiveRangeExtension), "F"},
+   {"enableTrampolines",                  "O\tenable generation of trampolines for method dispatch", SET_OPTION_BIT(TR_EnableTrampolines), "F"},
```

Comment:
I find the presence of this option in its currently named form confusing.  Every other platform X86, Power, ARM, Z (to some extent) already require trampolines as a matter of correctness.  Enablement isn't really an option, its a requirement.  I don't see the existing platforms starting to use this option because I'm not sure how it would be applied and what the semantics would be, which is why I think it needs to be clarified and made specific to the context in which you need these trampolines generated (zOS).  If I'm not thinking about this the right way can you please clarify?

I do think a context where trampolines are always generated at a call site would be useful, but that's more along the lines of your other `stressTrampoline` option.